apps/web/vibes/soul/primitives/tooltip/index.tsx

```diff
@@ -0,0 +1,62 @@
+import * as TooltipPrimitive from '@radix-ui/react-tooltip';
+import { clsx } from 'clsx';
+
+interface TooltipProps extends React.PropsWithChildren {
+  align?: 'center' | 'end' | 'start';
+  className?: string;
+  delayDuration?: number;
+  skipDelayDuration?: number;
+  trigger?: React.ReactNode;
+  open?: boolean;
+  setOpen?: (open: boolean) => void;
+  side?: 'top' | 'right' | 'bottom' | 'left';
+  sideOffset?: number;
+}
+
+// eslint-disable-next-line valid-jsdoc
```

Comment:
Should we remove this eslint-disable in VIBES?

Reply:
Ope yep! Good call